### PR TITLE
Challenge1 Task Completion

### PR DESCRIPTION
**Pull request by muthash**
**What does this PR do?**
Have user interfaces built with HTML/CSS using ​Bootstrap in WeConnect RESTful API

**Description of Tasks to be completed**
The interfaces capture:
1. User registration
2. User login
3. User password_reset request
4. A page where an authenticated user can register his/her business.
5. A page that shows the profile of a business and shows the available reviews from users about that business.
6. A page where a user updates his/her business profile
7. A business catalog page that allows viewers to search for businesses

**What are the relevant pivotal tracker stories**
#155547400
#155522779
#155553258
#155522751
#155522607
#155522722
#155522496

### CODE REVIEW

**Review by peterpaints — CHANGES_REQUESTED** [review]
Could you fix the indentation? Most text editors have an auto indent feature that should help

File: designs/UI/register.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                        </div>
+                        <small id="PasswordInput" class="form-text text-muted" style="text-align: center !important;">
+                            Already Registered? <a href="signin.html">Sign In </a>
+                        </small>					
+                    </form>	
+                    </div>
+                </div>
+                </div>
+                </div>
+        <section>
```
Comment (peterpaints) [inline designs/UI/register.html]: Did you mean for this section tag to be a closing one? As in </section> ?

File: designs/UI/password_reset.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Contact</a>
+                        </li>
+                    </ul>
+                    <form class="form-inline my-2 my-lg-0">
+                        <input class="form-control mr-sm-2" type="search" placeholder="Search WeConnect" aria-label="Search">
+                    </form>
+                    <div class="navbar-nav">
+                        <a class="nav-item nav-link" href="signin.html">Sign in </span></a>
```
Comment (peterpaints) [inline designs/UI/password_reset.html]: Missing <span> tag here.

**Review by peterpaints — CHANGES_REQUESTED** [review]
Your code looks good. The pages look good. Fix those small issues I have requested, then I'll merge. Once I merge you can move on.

**Review by peterpaints — APPROVED** [review]

**Review by peterpaints — CHANGES_REQUESTED** [review]
Changes look good. Now, once 2 of your peers have reviewed, I'll merge it.

**Comment by JoshuaOndieki** [comment]
I think the link explanations you've given on the homepage would be better placed in a FAQs page.
Alternatively, you could provide tooltips for the respective link buttons.
Great signup form. I commend you for those input tips.
According to my screen size, I have to scroll down to access the full sign up form. I suggest you make the form short. As a user, the first impression if I can't see the signup button for the form, I assume it's a long form. :tired_face:
Apart from the few points I've mentioned, you sure did a great job on your UIs.  :clap: :clap: :clap:

:sparkles: *Tip* :sparkles: 
Integrate Code climate to help you review your code standards.

**Comment by mmosoroohh** [comment]
It's long code but looks great. Well done indentation thus making the code look visible and clear.

**Comment by muthash** [comment]
@JoshuaOndieki Thanks for the feedback. In production the sitemap links in the index page will not be there. I inserted them to guide in the review instead of having to enter the dashboard links manually in the browser since the pages are not functional at the moment.

**Pull request merged**